rest-api-spec/src/main/resources/rest-api-spec/test/search/110_field_collapsing.yml

```diff
@@ -376,6 +376,7 @@ setup:
       reason: Field aliases were introduced in 6.4.0.
   - do:
       indices.put_mapping:
+        include_type_name: false
```

Comment:
I'm curious why this is needed since in 7.0 this is the default and in mixed version 6.x/7.x tests we are injecting `include_type_name: false`?

Reply:
I just saw in a different commit that we only inject the parameter for the create index API so that explains why we need this here